assets/js/react/components/common/login/index.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import {
+  FormControl,
+  InputLabel,
+  FilledInput,
+  InputAdornment,
+  IconButton,
+} from '@material-ui/core';
+import Visibility from '@material-ui/icons/Visibility';
+import VisibilityOff from '@material-ui/icons/VisibilityOff';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const Login = ({ username, onUsernameChange, password, onPasswordChange }) => {
+  const [isPasswordVisible, setIsPasswordVisible] = React.useState(false);
+
+  const toggleIsPasswordVisible = () => {
+    setIsPasswordVisible(!isPasswordVisible);
+  };
+  const handleMouseDownPassword = (event) => {
+    event.preventDefault();
+  };
+  const handleUsernameChange = (event) => {
+    onUsernameChange(event.target.value);
+  };
+  const handlePasswordChange = (event) => {
+    onPasswordChange(event.target.value);
+  };
+
+  return (
+    <Wrapper>
+      <FormControl variant="filled">
+        <InputLabel htmlFor="input-with-icon-adornment">
+          <FormattedMessage id="Username" />
```

Comment:
Il faudrait unifier avec l'autre form de sign in plus haut ?
"email" et pas le "username" que tout le monde oubli...